Description: Patch Makefile.linux
 Modify upstream linux Makefile to allow hardening/autobuilding
Author: Daniel Lintott <daniel@serverb.co.uk>

--- vpcs-0.5b0.orig/src/Makefile.linux
+++ vpcs-0.5b0/src/Makefile.linux
@@ -1,12 +1,11 @@
 RM=rm -f
-CC=gcc $(CCOPT)
+CC=gcc
 OSTYPE=Linux
-CPUTYPE=i386
 OPT=-O2
 HVOPT=-DHV
 
-CFLAGS=-D$(OSTYPE) -D$(CPUTYPE) $(OPT) $(HVOPT) -Wall -I. -DTAP
-LDFLAGS=-lpthread -lutil -s -static
+CFLAGS?=-D$(OSTYPE) -D$(CPUTYPE) $(OPT) $(HVOPT) -Wall -I. -DTAP
+LDFLAGS?=-lpthread -lutil -s -static
 OBJS=vpcs.o \
 	daemon.o \
 	readline.o \
@@ -31,10 +30,10 @@ OBJS=vpcs.o \
 all: vpcs
 
 vpcs: $(OBJS)
-	$(CC) $(OBJS) -o vpcs $(LDFLAGS) 
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -o vpcs $(LDFLAGS) 
 
 .c.o: keys.h vpcs.h packets.h dhcp.h
-	$(CC) $(INCLUDE_PATH) $(CFLAGS) -c $<
+	$(CC) $(INCLUDE_PATH) $(CPPFLAGS) $(CFLAGS) -c $<
 
 clean:
 	$(RM) *.o vpcs
